src/main/java/plates/PlateCell.java

```diff
@@ -1,76 +1,71 @@
 package plates;
 
-import java.util.Objects;
+import java.util.Arrays;
 
 public class PlateCell {
 
     private static final char[] LETTERS = {'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S',
             'T', 'U', 'V', 'W', 'X', 'Y', 'Z'};
 
     public Dimensions dimensions;
-    String cell;
     int row, column;
-    int cellNumber;
 
     public PlateCell(String cell, Dimensions dimensions) {
-        this.cell = cell;
+        String[] split = cell.split("(?<=\\D)(?=\\d)");
         this.dimensions = dimensions;
+        this.row = new String(LETTERS).indexOf(split[0]);
+        this.column = Integer.parseInt(split[1]) - 1;
+        if (column < 0) {
+            throw new IllegalArgumentException(cell + " cell does not exist!");
+        }
+        if (row + 1 > dimensions.getNumberOfRows() | column + 1 > dimensions.getNumberOfColumns()) {
+            throw new IllegalArgumentException(cell + " cell does not exist in " + dimensions.getSizeOfPlate() + " plate.");
+        }
     }
 
     public PlateCell(int row, int column, Dimensions dimensions) {
+        if (row <= 0 | column <= 0) {
```

Comment:
You're duplicating checks. Note, that you can invoke one constructor from another using `this(blah);`